resources/views/service/sponsors/index.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('service.layouts.app')
+
+@section('content')
+<div id="container">
+
+    @include('service.includes.sidebar')
+
+    <div id="main-content">
+        <h1>Sponsors</h1>
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>RVID Prefix</th>
```

Comment:
This isn't the `RVID Prefix`, that's for family registrations. This should probably be `Voucher Prefix`